libraries/AP_Camera/AP_Camera_Servo.h

```diff
@@ -29,12 +29,20 @@ class AP_Camera_Servo : public AP_Camera_Backend
     // Constructor
     using AP_Camera_Backend::AP_Camera_Backend;
 
+    void init() override;
```

Comment:
let's put the init lower like it is in other drivers.  Also if possible can we put a comment above it? I know it's pretty obvious what "init" is for but all our other methods have comments so I'd like to keep it consistent